content-repo/extra-docs/releases/20.5.2.md

```diff
@@ -0,0 +1,269 @@
+  - DISCLAIMER: Please consult with your legal team before implementing this playbook.
+  **Source:** 
+    - http://leginfo.legislature.ca.gov/faces/codes_displaySection.xhtml?lawCode=CIV&sectionNum=1798.82.
+- __Digital Guardian Demo Playbook__  
+This playbook will show how to handle an exfiltration event through Digital Guardian by emailing a user's manager and adding the user to a DG Watchlist.
+- __US - Breach Notification__  
+This playbook is triggered by a breach notification incident and then proceeds to the breach notification playbook for the relevant state.
+
+####  10 Improved Playbooks
+- __TIM - QRadar Add IP Indicators__  
+Fixed task condition.
+- __TIM - Run Enrichment For Hash Indicators__  
+Fixed input name.
+- __TIM - Process Indicators - Manual Review__  
+Fixed a typo.
+- __TIM - Run Enrichment For IP Indicators__  
+Fixed input name.
+- __TIM - Run Enrichment For Domain Indicators__  
+Fixed input name.
+- __TIM - Run Enrichment For Url Indicators__  
+Fixed input name.
+- __ExtraHop - Ticket Tracking__  
+Searching incidents is now done using SearchIncidentsV2 instead of the deprecated SearchIncidents automation.
+- __Email Address Enrichment - Generic v2.1__  
+Added a check that will prevent empty email addresses from being 
+enriched.
+- __URL Enrichment - Generic v2__  
+The playbook will not stop if Rasterize fails. This improves the playbook stability when rasterizing URLs of websites that are currently down.
+- __Phishing Investigation - Generic v2__  
+  - The playbook now uses Block Indicators - Generic v2 to block malicious indicators (off by default).
+  - Replaced the deprecated SendEmail automation with the send-mail command.
+
+---
+### Incident Fields
+Added incident fields for:
+- __Digital Guardian__  
+- __Logz.io__  
+- __US Breach Notification__  
+
+
+---
+### Layouts
+
+####  4 New Layouts
+- __Digital Guardian Security Event - Summary__  
+- __US Breach Notification - Summary__  
+- __Illusive Networks Incident - Summary__  
+- __Logz.io Alert - Summary__  
+
+
+####  4 Improved Layouts
+- __domainRepUnified - Indicator Details__  
+- Fixed an issue where Extended Details would not display the *Threat Category* and *Threat Category Confidence* fields.
+- __ipRep - Indicator Details__  
+- Fixed an issue where Extended Details would not display the *Threat Category* and *Threat Category Confidence* fields.
+- __unifiedFileRep - Indicator Details__  
+Fixed an issue where Extended Details would not display the *Threat Category* and *Threat Category Confidence* fields.
+- __urlRep - Indicator Details__  
+Fixed an issue where Extended Details would not display the *Threat Category* and *Threat Category Confidence* fields.
+
+---
+### Classification & Mapping
+
+####  3 New Classification & Mapping
+- __Logz.io__  
+- __Digital Guardian__  
+- __IllusiveNetworks__  
+
+---
+### Reputations
```

Comment:
Not clear what this section is. It looks like it contains garbage.

Reply:
I think we can remove this whole section. @kirbles19 What do you think?

Reply:
Which, Reputations?

Reply:
yeap

Reply:
I'll take it out of my version